src/cpp/FilesInformation/Loaders/FilesLoader.cpp

```diff
@@ -7,7 +7,7 @@
 
 Directory *FilesLoader::loadDirectory(const std::string &filepath) {
     Directory *directory = new Directory(filepath);
-    for (const auto &entry: std::__fs::filesystem::directory_iterator(filepath)) {
+    for (const auto &entry: std::filesystem::directory_iterator(filepath)) {
         if (entry.is_directory()) {
             directory->add(loadDirectory(entry.path()));
```

Comment:
According to C++17 standard, std::filesystem::directory_entry::path() returns path object, not std::string, and there is no implicit conversion to string. Please implement explicit conversion (e.g. using std::filesystem::path::string() or one of alternatives)

Reply:
This is an interesting moment because I am using compiler for C++17 to test everything and it should have notified me about that, but it didn't. Do you have any thoughts on why it didn't?

Reply:
Some compilers may implement additional methods that are not specified by standard - e.g. direct conversion from path to string. However, if you want your code to be portable - you should check which capabilities are specified by standard (e.g. using cppreference.com). Unfortunately there seems to be no easy way to test it - although you may try compiling your code with different compilers (gcc, clang, msvc) and see if there are any differences.